gremlin-javascript/example/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "GLV example for JavaScript with Node",
+  "main": "example.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Feel free to put your own name here if you would like, or you could also copy `"author": "Apache TinkerPop team",` from the gremlin-javascript package.json. Leaving the author blank should be fine as well.

Reply:
Put name in field